packages/mobile/src/screens/edit-track-screen/EditTrackScreen.tsx

```diff
@@ -28,7 +28,6 @@ const EditTrackSchema = Yup.object().shape({
         .positive()
         .min(0.99, 'Price must be at least $0.99.')
         .max(9999.99, 'Price must be less than $9999.99.')
```

Comment:
Do these still error if it is present and outside the range?

Reply:
they do if outside range, it doesnt error if it's empty i believe. all this is still behind a feature flag anyway, so hoping to come back to this validation, likely just take the zod stuff from web